runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -129,6 +124,48 @@ public void onConfigChanged(ConfigChanged config)
 		}
 	}
 
+	// Use a high priority so that this is called way after any other plugins add menu entries
+	@Subscribe(priority = 999_999_999)
```

Comment:
This is overkill. [Of hub plugins which subscribe to MenuOpened](https://abextm.github.io/plugin-hub/#WyJeTG5ldC9ydW5lbGl0ZS9hcGkvZXZlbnRzL01lbnVPcGVuZWQiLCIiXQ==), only the following set priorities:

* **no-examine**: `-1` (https://github.com/Skretzo/runelite-plugins/blob/f97c0ea114e7a570c0e1857c1598999ee3d0c044/src/main/java/com/noexamine/NoExaminePlugin.java#L45-L48)
* **weapon-charges-2**: `-1` (https://github.com/geheur/weapon-charges/blob/d9a102a49569dd5d67f0e8a2c218f0d4d79a283b/src/main/java/com/weaponcharges/WeaponChargesPlugin.java#L272-L280)
* **hotkeyable-menu-swaps**: `-1` (https://github.com/geheur/More-menu-entry-swaps/blob/a4b8de2ed385b77c421df204f247d596f39e88c9/src/main/java/com/hotkeyablemenuswaps/HotkeyableMenuSwapsPlugin.java#L595-L596)

So with that in mind, this priority only needs to be higher than that of other core plugins.